daemons/ipa-kdb/ipa_kdb_kdcpolicy.c

```diff
@@ -56,6 +56,8 @@ ipa_kdcpolicy_check_as(krb5_context context, krb5_kdcpolicy_moddata moddata,
     
     /* If no mechanisms are set, allow every auth method */
     if (ua == IPADB_USER_AUTH_NONE) {
+	/* Set lifetime_out to a random value in the last hour of lifetime, between 23 and 24 hours */
+        *lifetime_out = (86400 - IPA_KRB_JITTER_WINDOW) + (rand() % IPA_KRB_JITTER_WINDOW);
```

Comment:
Might be worth using getrandom() or OSSL_rand() here?  rand() is fine for this application, but static analyzers are going to complain later :)

Reply:
I used getrandom(), but can change to RAND_bytes() if anyone thinks that's better.